service/src/main/kotlin/fi/espoo/evaka/application/ApplicationQueries.kt

```diff
@@ -115,6 +115,35 @@ fun duplicateApplicationExists(
         .isNotEmpty()
 }
 
+fun activePlacementExists(
+    h: Handle,
```

Comment:
You shouldn't use `Handle` directly anymore, prefer `Database.Read` and `Database.Transaction`.